docs/DEVELOPMENT.md

```
@@ -0,0 +1,1015 @@
+  
+  - repo: https://github.com/pycqa/isort
+    rev: 5.12.0
+    hooks:
+      - id: isort
+```
+
+## Code Structure
+
+### Backend Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)